sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1541,9 +1568,20 @@ def __setitem__(self, key, value):
     else:
       raise NotImplementedError(key)
 
+  @frame_base.with_docs_from(pd.DataFrame)
   @frame_base.args_to_kwargs(pd.DataFrame)
   @frame_base.populate_defaults(pd.DataFrame)
   def align(self, other, join, axis, copy, level, method, **kwargs):
+    """Aligning per-level is not yet supported. Only the default,
```

Comment:
"per-level" -> "per level" (because it's technically being used as an adverb, and ... English is weird)

Reply:
Done, thanks!